test/client-test.js

```diff
@@ -43,6 +43,7 @@ describe('Client', () => {
       it('returns SMART OAuth URIs', async function () {
         nock(this.baseUrl)
           .matchHeader('accept', 'application/json+fhir')
+          .matchHeader('Authorization', '')
```

Comment:
`.matchHeader('Authorization', undefined)` seems like it would be more "correct" -- but this only works if an empty String is specified.

Reply:
we should not set matchHeader at all here.